qt/scientific_interfaces/Indirect/CMakeLists.txt

```diff
@@ -81,17 +74,24 @@ set(SRC_FILES
     IndirectSettingsView.cpp
     IndirectSimulation.cpp
     IndirectSimulationTab.cpp
-    IndirectSqw.cpp
-    IndirectSqwModel.cpp
-    IndirectSqwView.cpp
-    IndirectSymmetrise.cpp
-    IndirectSymmetriseView.cpp
-    IndirectSymmetriseModel.cpp
     IndirectTab.cpp
     IndirectTools.cpp
     IndirectToolsTab.cpp
     IndirectTransmission.cpp
     IndirectTransmissionCalc.cpp
+    InelasticDataManipulation.cpp
```

Comment:
Do we want these files moved into a separate scientific interfaces folder, e.g. `qt/scientific_interfaces/Inelastic/`, because they are not necessarily belonging to `Indirect` anymore?

Reply:
while these interfaces are all inelastic they inherit from the base IndirectTab object. so for now it makes more sense to have them together.